backend/fms_core/filters.py

```diff
@@ -0,0 +1,58 @@
+from django.db.models import Q
+
+from .models import Container, Sample, Individual
+
+import django_filters
+
+from .viewsets._constants import (
+ _container_filterset_fields,
+ _sample_filterset_fields,
+ _individual_filterset_fields,
+)
+
+class ContainerBatchFilter(django_filters.FilterSet):
```

Comment:
I would not define the whole filterset by the case of the 2 fields that are batched ... I would name it ContainerFilter ... unless you plan to have more than one filterset for each model.